lib/rubocop/cop/lint/float_comparison.rb

```diff
@@ -81,21 +81,15 @@ def literal_safe?(node)
           (node.numeric_type? && node.value.zero?) || node.nil_type?
         end
 
-        # rubocop:disable Metrics/PerceivedComplexity
         def check_send(node)
           if node.arithmetic_operation?
             float?(node.receiver) || float?(node.first_argument)
           elsif FLOAT_RETURNING_METHODS.include?(node.method_name)
             true
           elsif node.receiver&.float_type?
-            if FLOAT_INSTANCE_METHODS.include?(node.method_name)
-              true
-            else
-              check_numeric_returning_method(node)
-            end
+            FLOAT_INSTANCE_METHODS.include?(node.method_name) || check_numeric_returning_method(node) # rubocop:disable Layout/LineLength
```

Comment:
Instead of disabling `Layout/LineLength`, split to multiple lines:
```suggestion
            FLOAT_INSTANCE_METHODS.include?(node.method_name) ||
              check_numeric_returning_method(node) # rubocop:disable Layout/LineLength
```

Reply:
@dvandersluis fixed